tensorflow/python/autograph/impl/api_test.py

```diff
@@ -289,6 +289,21 @@ def test_fn(x, y, z):
         options=DEFAULT_RECURSIVE)
     self.assertEqual((1, 2, 3), self.evaluate(x))
 
+  @test_util.run_v1_only('b/120545219')
+  def test_converted_call_functools_partial_decorated(self):
+
+    api.converted_call(options=DEFAULT_RECURSIVE)
```

Comment:
This statement has no effect - perhaps you meant it to be a decorator?
At any rate, converted_call is not intended to be a decorator, but you can still use it as x = api.converted_call(test_call_partial, ...)

Reply:
Yes sorry was a wrong cut and paste from another experiment. It was `api.convert(recursive=DEFAULT_RECURSIVE)`

Reply:
Got it, noprob. I think we still need the `@` to make it a decorator: `@api.convert(...)`?

Reply:
Yes i fixed the typo